src/main/java/io/debezium/connector/cassandra/CassandraConnectorConfig.java

```diff
@@ -160,6 +160,7 @@
 
     public static final Field COMMIT_LOG_RELOCATION_DIR = Field.create("commit.log.relocation.dir")
             .withType(Type.STRING)
+            .withValidation(Field::isRequired)
```

Comment:
@gunnarmorling Any idea why `.withValidation(Field::isRequired)` here didn't work as expected please? We tested the change locally with the validation here by not passing in this config, but the validation check didn't fail..

Reply:
Good question, @bingqinzhou. From a quick glance at `CassandraConnectorTask`, I'm not sure whether _any_ property validation is done really for the Cassandra connector? For the other connectors it's done by/via Kafka Connect, but for the Cassandra one we'd have to do it ourselves in the Cassandra connector code, until we've moved it over to Debezium Server as we discussed a few weeks ago (for reference, [here](https://github.com/debezium/debezium/blob/master/debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java#L658-L660) is what we do for the embedded engine).